### ISSUE TITLE
why code need retrain is not identity as the paper

### ISSUE THREAD

**Issue by lndawn** [friendshipkim/neuron-merging#4]
I read your paper and the innovation is one-shot and data-free get the better results than only pruning ,so I very interesting your work.when I download the code then run and find it will enter the train.so I want to know how to configure the args to get the model the no retrain

**Comment by friendshipkim** [comment]
Thanks for your interest in our work. 

If you do not want to train the model and just use the pretrained model, use `retrain` and `evaluate` args at the same time. Please refer to the files in `./scripts/`

**Comment by lndawn** [comment]
thanks a lot,my question is solved

**Comment by lndawn** [comment]
Hi,there's another problem,the prune and merge get same accuracy when I close retrain,the result is not identity as the paper,the loss is bigger than your experiment result
![image](https://user-images.githubusercontent.com/14303955/127089062-823c96d8-3e29-4108-8193-59585b8e24c0.png)
![image](https://user-images.githubusercontent.com/14303955/127089123-38df4963-f3af-4d7c-b9f0-ebf027df995a.png)

**Comment by lndawn** [comment]
I set params  --treshhold,the merge result is better than prune,but still loss a lot accuracy
